docs/sources/access/index.md

```diff
@@ -0,0 +1,67 @@
+---
+description: Access or install Explore Profiles.
+canonical: https://grafana.com/docs/grafana/latest/explore/simplified-exploration/profiles/access/
+keywords:
+  - Install
+  - Configure
+  - Explore Profiles
+title: Access or install Explore Profiles
+menuTitle: Access or install
+weight: 150
+refs:
+  pyroscope-data-source:
+    - pattern: /docs/grafana/
+      destination: /docs/grafana/<GRAFANA_VERSION>/datasources/pyroscope/
+    - pattern: /docs/grafana-cloud/
+      destination: /docs/grafana-cloud/connect-externally-hosted/data-sources/pyroscope/
+---
+
+# Access or install Explore Profiles
+
+Explore Profiles is a native Grafana application designed to integrate seamlessly with [Pyroscope](https://github.com/grafana/pyroscope), the open source continuous profiling platform, providing a smooth, query-less experience for browsing and analyzing profiling data.
+
+You can use Explore Profiles in Grafana Cloud or with Grafana open source stand-alone.
+
+## Before you begin
+
+To use Explore Profiles with Grafana Cloud, you need:
+
+* A Grafana Cloud account
+* A Grafana stack in Grafana Cloud with a configured Hosted profiles or [Pyroscope data source](ref:pyroscope-data-source)
+
+To use Explore Profiles with Grafana open source, you need:
+
+* Your own Grafana instance
+* A configured [Pyroscope data source](ref:pyroscope-data-source)
+* The [Explore Profiles plugin](https://grafana.com/grafana/plugins/grafana-pyroscope-app/)
+
+## Install the Explore Profiles plugin
+
+Explore Profiles is distributed as a Grafana Plugin. You can find it in the official [Grafana Plugin Directory](https://grafana.com/grafana/plugins/grafana-pyroscope-app/).
+
+### Install in [Grafana Cloud](https://grafana.com/products/cloud/)
+
+All Grafana Cloud instances come with the Explore Profiles plugin preinstalled.
+
+### Install in your Grafana instance
+
+You can install Explore Profiles in your own Grafana instance using `grafana-cli`:
+
+```shell
+grafana-cli plugins install grafana-pyroscope-app
+```
+
+Alternatively, follow these steps to install Explore Profiles in Grafana:
+
+1. In Grafana, go to **Administration** > **Plugins and data** > **Plugins**.
+2. Search for "Explore Profiles".
+3. Select Explore Profiles.
+4. Click **Install**
+
+### Install using environment variables
+
+If you want to install the app in a Docker container, you need to configure the following environment variable:
+
+```shell
+GF_INSTALL_PLUGINS=grafana-pyroscope-app
+```
```

Comment:
I do think you need to also enable the plugin in a second step. I don't know why this is required. When I install for Example "Explore Logs" it is there right away.

![image](https://github.com/user-attachments/assets/24d1c6cf-a9de-4ca4-8ee2-eb029c9569e5)

I actually can't find where to do this, the enable seems to be missing for us too.

![image](https://github.com/user-attachments/assets/8e1c6ce3-2568-4344-be5f-e7264bc456d9)

cc @grafakus

Reply:
Ok this PR should enable the auto install https://github.com/grafana/explore-profiles/pull/65

Reply:
Just to verify: With your PR merged, a user can install thel plugin and it's automatically enabled? Or do we need to add a step to Enable?

Reply:
@knylander-grafana Ideally the PR in #65 will fix it so we do not require changes to the docs, but @petethepig will verify that.

Reply:
I'm out tomorrow, but if things look good, please give an approval and feel free to merge the docs.